packages/hooks/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  printWidth: 100,
+  semi: true,
+  singleQuote: true,
+  trailingComma: 'es5',
+  jsxSingleQuote: true,
+};
```

Comment:
Great suggestion. I think this can be coupled with some other devops-y changes @etr2460 suggested in his reviews. We can batch these change together in a PR dedicated to that.